### PR TITLE
[luci-interpreter] Support int8 conv2d 

### PR DESCRIPTION
**Pull request by binarman**
This PR enables int8 quantizer depthwise conv2d.

Related issue: #7677 

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

**Comment by binarman** [comment]
"NO MERGE" because this PR uses some test-related code from #7679, so I want to merge it first

**Comment by binarman** [comment]
@nnfw-bot test nncc-release

**Comment by binarman** [comment]
@nnfw-bot test nncc-release

**Comment by binarman** [comment]
@nnfw-bot test nncc-release

**Comment by binarman** [comment]
@nnfw-bot test nncc-release

File: compiler/luci-interpreter/pal/linux/PALConv2d.h [hunk 1]
```diff
@@ -57,6 +58,22 @@ static inline void Conv(const tflite::ConvParams &params, const tflite::RuntimeS
...
+                                  int8 *output_data, const tflite::RuntimeShape &im2col_shape,
+                                  int8 *im2col_data)
+{
+  (void)im2col_shape;
+  (void)im2col_data;
+  // TODO enable optimized version
+  tflite::reference_integer_ops::ConvPerChannel(params, mult, shifts, input_shape, input_data,
+                                                filter_shape, filter_data, bias_shape, bias_data,
+                                                output_shape, output_data);
+}
```
Comment (BalyshevArtem) [inline compiler/luci-interpreter/pal/linux/PALConv2d.h:75]: Why can't we support the optimized version now? We can add 
`#include <tensorflow/lite/kernels/internal/optimized/integer_ops/conv.h>` and then change 
```suggestion
  tflite::optimized_integer_ops::ConvPerChannel(params, mult, shifts, input_shape, input_data,
                                                filter_shape, filter_data, bias_shape, bias_data,
                                                output_shape, output_data, im2col_shape, im2col_data);
}
```

Comment (binarman) [reply compiler/luci-interpreter/pal/linux/PALConv2d.h:75]: We can, but this requires linking with several new libraries from tensorflow, so I postponed it =)

**Review by BalyshevArtem — APPROVED** [review]
LGTM

File: compiler/luci-interpreter/src/kernels/Conv2D.test.cpp [hunk 2]
```diff
@@ -283,6 +283,78 @@ TEST(Conv2DTest, Uint8_CWQ)
...
+    1, 1, 1, 1, // row = 1
+    2, 2, 2, 2, // row = 2
+                // Second batch
+    1, 2, 3, 4, // row = 1
+    1, 2, 3, 4, // row = 2
+  };
+  std::vector<float> filter_data{
+    1,  2,  3,  4, // first 2x2 filter
+    -1, 1,  -1, 1, // second 2x2 filter
+    -1, -1, 1,  1, // third 2x2 filter
```
Comment (AShedko) [inline compiler/luci-interpreter/src/kernels/Conv2D.test.cpp:300]:
```suggestion
    1,  2, 3, 4,  // first 2x2 filter
    -1, 1, -1, 1, // second 2x2 filter
    -1, -1, 1, 1, // third 2x2 filter
```
Alignment?

**Review by AShedko — APPROVED** [review]
LGTM

**Comment by binarman** [comment]
Thank you! =)

**Pull request merged**